libs/financial-aid/auth/src/lib/auth.module.ts

```diff
@@ -0,0 +1,42 @@
+import { DynamicModule, Global } from '@nestjs/common'
+import { PassportModule } from '@nestjs/passport'
+import { JwtModule } from '@nestjs/jwt'
+
+import { JwtStrategy } from './jwt.strategy'
+import { SharedAuthService } from './auth.service'
+import { COOKIE_EXPIRES_IN_SECONDS } from '@island.is/financial-aid/shared'
+
+export interface SharedAuthModuleOptions {
+  jwtSecret: string
+  secretToken: string
+}
+
+@Global()
+export class SharedAuthModule {
```

Comment:
This is a lot of security-related duplicated code.

Can you use `libs/auth-nest-tools` and `libs/auth/react` instead?

Reply:
I had a meeting with a developer from Fuglar yesterday and are having a meeting with the PO of the identity server next week. To figure out when its going to be a good time for us to migrate over.

Reply:
Ok sure. I'd like to know more, but I guess this can wait till fall :)